app/assets/stylesheets/application.css.scss

```diff
@@ -15,6 +15,38 @@
  *= require font-awesome
  */
 
+#map-container { 
+  border-color: #fff;
+  border-radius: 16px 16px;
```

Comment:
Shorthand form for property `border-radius` should be written more concisely as `16px` instead of `16px 16px`